### PR TITLE
Add Deepgram TTS and remove ai-jsx dependency

### PR DESCRIPTION
**Pull request by juberti**
ASR/TTS code has been ported directly into voice-demo, for easier maintenance.

### CODE REVIEW

File: src/app/asr/asr.tsx [hunk 1]
```diff
@@ -0,0 +1,942 @@
...
+    tokenFunc: GetToken,
+    language?: string,
+    model?: string,
+  ) {
+    super("deepgram", manager, tokenFunc, language, model);
+  }
+  async start() {
+    this.buf = "";
+    const params = new URLSearchParams({
+      model: this.model ?? "nova-2-ea",
```
Comment (mdwelsh) [inline src/app/asr/asr.tsx]: Should just be "nova-2" at this point

**Review by mdwelsh — APPROVED** [review]
I assume this is all just moving code around so I didn't review it super carefully. Wondering why this would be in the voice-demo repo and not part of the Fixie SDK instead?

**Comment by juberti** [comment]
This code is only for demo and testing purposes now, so I think it fits better in voice-demo. It's not connected to the Fixie service in any way.

File: src/app/asr/asr.tsx (hunk 1 above)
Comment (juberti) [reply src/app/asr/asr.tsx]: ah good catch!

**Pull request merged**